draft-ietf-sframe-enc.md

```diff
@@ -548,7 +548,10 @@ follows, where `Nk` represents the key size for the AES block cipher in use and
 
 ~~~~~
 def derive_subkeys(sframe_key):
-  aead_secret = HKDF-Extract(sframe_key, 'SFrame10 AES CTR AEAD')
+  # tag_len = length of a tag for this cipher, in bytes
```

Comment:
Where is it that this is intended to be defined? Perhaps it doesn't matter in this example?

Reply:
It's the value `Nt` in the IANA table.